docs/14_0_0/components/datatable.md

```diff
@@ -1006,67 +1006,59 @@ JPALazyDataModel<MyEntity> lazyDataModel = JPALazyDataModel.<MyEntity>builder())
         .build();
 ```
 
-If you have selection enabled, you can either pass the rowKey via JPA metamodel:
+If you have selection enabled, we need a `rowKey`. 
+It's very likely that this is the mapped JPA `@Id`, so we try to auto-lookup per default.
+Otherwise you can also define it:
 
 ```java
 JPALazyDataModel<MyEntity> lazyDataModel = JPALazyDataModel.<MyEntity>builder())
-        .entityClass(MyEntity.class)
-        .entityManager(() -> entityManager)
-        .rowKeyField(MyEntity_.id)
-        .build();
-```
-
-or as plain String:
-
-```java
-JPALazyDataModel<MyEntity> lazyDataModel = JPALazyDataModel.<MyEntity>builder())
-        .entityClass(MyEntity.class)
-        .entityManager(() -> entityManager)
-        .rowKeyField("id")
-        .build();
+        ...
+        .rowKeyField(MyEntity_.id) // JPA MetaModel
+        .rowKeyField("id") // or as String
+        ...
 ```
 
 or provide a existing JSF `Converter`:
 
 ```java
 JPALazyDataModel<MyEntity> lazyDataModel = JPALazyDataModel.<MyEntity>builder())
-        .entityClass(MyEntity.class)
-        .entityManager(() -> entityManager)
+        ...
         .rowKeyConverter(myConverter)
-        .build();
+        ...
 ```
 
-or enable case insensitive searching:
+#### Case insensitive searching
+Per default the filters are case sensitive but you can also disable it:
 
 ```java
 JPALazyDataModel<MyEntity> lazyDataModel = JPALazyDataModel.<MyEntity>builder())
-        .entityClass(MyEntity.class)
-        .entityManager(() -> entityManager)
+        ...
         .caseSensitive(false)
-        .build();
+        ...
 ```
 
-or enable wildcard support for characters `*`, `%`, `?` and `_`. Characters `*` and `%` means any characters 
+#### Wildcards
+Enable wildcard support for characters `*`, `%`, `?` and `_`. Characters `*` and `%` means any characters 
 while `?` and `_` mean replace a single character.  For example `Smith*` would find all matches starting with 
 the word `Smith`. For single character replacement like `Te?t` would match words `Tent` and `Test`.
 
 ```java
 JPALazyDataModel<MyEntity> lazyDataModel = JPALazyDataModel.<MyEntity>builder())
-        .entityClass(MyEntity.class)
-        .entityManager(() -> entityManager)
+        ...
         .wildcardSupport(true)
-        .build();
+        ...
 ```
 
```

Comment:
Not a fan of all these paragraphs, cumbersome and not so great to read... I think it'd be better if you show every options with a table (name + description): the info would be straightforward to get. Also, I don't think you need to show every use case with code associated. Considering a builder is being used, it speaks by itself

Reply:
@Rapster i think in this case people need examples and show exactly what the features will do else we will get questions.

Reply:
Just to be clear, I meant a markdown table. I would simply describe that, there is a builder to instanciate a datamodel, give a very basic example, and then describe all the features users can enable contained in a md table (just my 2 cents tbh, builders are supposed to be self documented)